src/lib/controllers/WebhookController.js

```diff
@@ -31,15 +43,30 @@ class WebhookController extends Controller {
   @httpVerb('post')
   @route('/v1/webhooks')
   post(model: Webhook) {
-    const newWebhook = this._webhookRepository.create(model);
-    return this.ok({
-      created_at: newWebhook.created_at,
-      event: newWebhook.event,
-      hookUrl: settings.baseUrl + '/v1/webhooks/' + newWebhook.id,
-      id: newWebhook.id,
-      ok: true,
-      url: newWebhook.url,
-    });
+    try {
+      const validateError = validateWebhookModel(model);
+      if (validateError) {
+        throw validateError;
+      }
+
+      const isEventInUse =
```

Comment:
Actually... The official particle server allows you to have multiple webhooks for the same event.